src/http/mod.rs

```
@@ -70,14 +70,28 @@ pub fn min_ttl<'a>(res_vec: impl Iterator<Item = &'a Response>) -> i32 {
   min
 }
 
-fn log_launch(sc: &ServerConfig) {
+fn log_launch_and_open_browser(sc: &ServerConfig) {
   let addr = sc.addr().to_string();
   log::info!("🚀 Tailcall launched at [{}] over {}", addr, sc.http_version());
   if sc.graphiql() {
-    log::info!("🌍 Playground: {}", sc.graphiql_url());
+    let url = sc.graphiql_url();
+    log::info!("🌍 Playground: {}", url);
+
+    log::info!("🌍 Opening playground on your default browser");
+
+    open_browser(url);
   }
 }
 
+fn open_browser(graphiql_url: String) {
+  let opened = webbrowser::open_browser(webbrowser::Browser::Default, graphiql_url.as_str());
+
+  match opened {
+    Ok(_) => log::info!("{} opened on your browser", graphiql_url),
+    Err(e) => log::error!("Failed to open default browser, error: {}", e),
+  };
```

This should silently fail.